sauron/sauron.py

```diff
@@ -161,4 +173,23 @@ def getfeerate(plugin, **kwargs):
     "The URL of the esplora instance to hit (including '/api')."
 )
 
+plugin.add_option(
+    "sauron-use-tor",
+    False,
+    "Proxy requests through Tor.",
+    opt_type="flag"
+)
+
+plugin.add_option(
+    "sauron-tor-address",
+    "localhost",
+    "Tor's SocksPort address. Requires --sauron-use-tor."
+)
+
+plugin.add_option(
+    "sauron-tor-port",
+    9050,
+    "Tor's SocksPort port. Requires --sauron-use-tor."
+)
+
```

Comment:
These could be collapsed to a single option `--sauron-tor-proxy`? If it is not set we don't use TOR, if it is it's an `address[:port]` string that tells us where the SOCKS5 port is.